swan-lake/learn/quick-tour.md

```diff
@@ -12,14 +12,19 @@ redirect_from:
 
 ## Installing Ballerina
 
-1. [Download](https://ballerina.io/downloads) Ballerina based on the Operating System you are using. 
-1. Follow the instructions given on the [Getting Started](/swan-lake/learn/getting-started) page to set it up. 
-1. Follow the instructions given on the [Visual Studio Code Plugin](/swan-lake/learn/tools-ides/vscode-plugin) page or the [IntelliJ IDEA Plugin](/swan-lake/learn/tools-ides/intellij-plugin) page to set up your preferred editor for Ballerina.
+1. [Download](/downloads) Ballerina based on the operating system you are using. 
+1. [Install](/swan-lake/learn/installing-ballerina) Ballerina. 
+1. Set up [Visual Studio Code](/swan-lake/learn/tools-ides/vscode-plugin) for Ballerina.
 
-## Writing a Service, Running It, and Invoking It
+## Creating a Service and a Client 
 
-Write a simple Hello World HTTP service in a file with the `.bal` extension.
+Let's create a custom Ballerina HTTP service and also an HTTP client to invoke it.
```

Comment:
Let's remove the "custom" part from here.

Reply:
Done.